libr/bin/obj.c

```diff
@@ -264,6 +260,7 @@ R_API int r_bin_object_set_items(RBinFile *binfile, RBinObject *o) {
 }
 
 R_API RBinObject *r_bin_object_get_cur(RBin *bin) {
+	r_return_val_if_fail (bin, NULL);
```

Comment:
we should kill the concept of "current binobj"

Reply:
that's a good point i wanted to ask you indeed. what's the point in having multiple rbinobj in a rbinfile? I though as well we could remove this thing that just complicates stuff.